static/css/restyle/restyle.less

```diff
@@ -2579,12 +2579,18 @@ body.windows .install-shell .platform.mac {
   img {
     height: 1.1em;
     margin-right: 0.25em;
+    .html-rtl & {
+      margin-left: 0.25em;
```

Comment:
the margin/padding/whatever of the surrounding objects seems to make it unnecessary. e.g. https://addons.allizom.org/ar/firefox/addon/rainbow-color-tools - it's only gap between the icon and the text that's obviously needed.